src/pdfFiller/fillPdf.ts

```diff
@@ -78,6 +78,11 @@ export async function create1040 (): Promise<Uint8Array> {
 
   if (state.taxPayer !== undefined) {
     const f1040 = new F1040(state.taxPayer)
+
+    if (f1040.errors().length > 0) {
+      return await Promise.reject(f1040.errors())
```

Comment:
The await isn't required here

Reply:
Nevermind!

Reply:
Yea I didn’t either. Format on save just added it.